src/ir/module-utils.h

```diff
@@ -135,8 +135,12 @@ template<typename T>
 inline void renameFunctions(Module& wasm, T& map) {
   // Update the function itself.
   for (auto& pair : map) {
-    assert(!wasm.getFunctionOrNull(pair.second));
-    wasm.getFunction(pair.first)->name = pair.second;
+    if (Function* F = wasm.getFunctionOrNull(pair.first)) {
+      // In some cases t he function itself might have been removed already
```

Comment:
typo in `the`

also please move this comment to the outside, as it's part of the definition of the method, that we allow the function to not exist.

Reply:
Sorry, I wanted to land this to get the waterfall green again.  Will upload a followup

Reply:
No problem, yeah, I gree the urgency justified a quick response.